README.md

```diff
@@ -88,16 +88,13 @@ This project requires at least the following to build:
 
 * C++17
```

Comment:
```suggestion
* C++20
```

Below we're now defaulting to 20, which seems to be the right default - suggest we update here too.

Reply:
I didn't change this detail. for what it's worth.

I believe the library supports C++17 entirely, but certain test cases require C++20 to run.

I'm pretty sure we could backport the library to C++03, even. I think all of the build requirements are related to the unit tests, but I've been focusing mostly on tooling details, so I haven't followed up on this detail.

Given you've approved the PR, I'll assume that means we're OK addressing the C++ standard compatibility in a separate PR or issue.

Reply:
Bret is right here.

Reply:
This shouldn't affect this PR, but I was writing a convenience utility for processing json in cmake in a more declarative way instead of the convoluted string function that currently exists. Maybe unnecessary if we don't expect to write too much of this cmake code like this